### PR TITLE
Adding tpl-test

### PR DESCRIPTION
**Pull request by wiheto**
This is a template for documentation/testing purposes

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -37,3 +37,6 @@
 [submodule "tpl-UNCInfant"]
 	path = tpl-UNCInfant
 	url = https://github.com/templateflow/tpl-UNCInfant
+[submodule "tpl-test2"]
+	path = tpl-test2
+	url = ./tpl-test2
```
Comment (oesteban) [inline .gitmodules]:
```suggestion
[submodule "tpl-test"]
	path = tpl-test
	url = https://github.com/templateflow/tpl-test
```

File: .gitmodules [hunk 2]
```diff
@@ -31,6 +31,12 @@
...
 	url = https://github.com/templateflow/tpl-MNIInfant
+[submodule "tpl-test"]
+	path = tpl-test
+	url = https://github.com/templateflow/tpl-test
 [submodule "tpl-UNCInfant"]
 	path = tpl-UNCInfant
 	url = https://github.com/templateflow/tpl-UNCInfant
+[submodule "tpl-test2"]
+	path = tpl-test2
+	url = https://github.com/wiheto/
```
Comment (oesteban) [inline .gitmodules]: What's this for?

Comment (wiheto) [reply .gitmodules]: Oh sorry. That was me trying something else and shouldnt have been included. Will remove.

**Comment by oesteban** [comment]
Okay, I think we can deprecate this one. I'll let you know when the new submission process is finished and we can add the test template then.

**Pull request closed without merging**